splinterd/Cargo.toml

```diff
@@ -112,6 +113,7 @@ experimental = [
     "service-endpoint",
     "service-timer-interval",
     "service2",
+    "service2-echo",
```

Comment:
I think we should change the name of this feature to just `service-echo` or `echo`

Reply:
updated to `service-echo`